libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -615,7 +616,12 @@ bool AP_Arming::servo_checks(bool report) const
         check_passed = false;
     }
 #endif
-    
+
+    if (AP::internalerror().errors() != 0) {
```

Comment:
This is a system check, not a servo check....

Reply:
Fixed.